src/components/discount/Discount.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "../discount/Discount.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHandHoldingDollar } from "@fortawesome/free-solid-svg-icons";
+
+function Discount() {
```

Comment:
add {title, subTitle} as arguments
and replace with text so component will be reusable